CMakeLists.txt

```diff
@@ -597,7 +597,7 @@ if(WIN32)
   target_sources(lovr PRIVATE src/core/os_win32.c)
   target_sources(lovr PRIVATE etc/lovr.rc)
   if (MSVC)
-    set_target_properties(lovr PROPERTIES COMPILE_FLAGS "/wd4244 /MP")
+    set_target_properties(lovr PROPERTIES COMPILE_FLAGS "/wd4244 /wd4146 /wd4116 /MP")
```

Comment:
From Microsoft's description, I'm having trouble understanding what `C4116` actually is.  Was there somewhere this warning was appearing?

Since 4146 is sometimes useful, it might be good to silence the warning only in audio.c with `set_source_files_properties`.  Feel free to skip that though.

Reply:
Sorry, C4116 was relevant to util.c not audio.c. util.c has:

`uint32_t u = ((union { float f; uint32_t u; }) { f }).u;`

The union here is being used for punning, but because it is anonymous, Microsoft apparently believes that there is no way it could have an effect. Either they literally forgot unions existed or, more likely, they introduced this warning for C++ (where union type punning works differently) and then forgot to remove it for C11. Or maybe CMake isn't building properly in C11 mode to start with, I haven't tested.

I can do the set_source_files_properties thing.